dotnet/src/Functions/Functions.OpenApi/Extensions/OpenApiKernelExtensions.cs

```diff
@@ -356,7 +356,7 @@ private static string ConvertOperationIdToValidFunctionName(string operationId,
             Verify.ValidFunctionName(operationId);
             return operationId;
         }
-        catch (KernelException)
+        catch (ArgumentException)
```

Comment:
How about a comment explaining why this exception is being caught and ignored?

Reply:
Make sense, done.